support/crown/Cargo.toml

```diff
@@ -20,3 +20,7 @@ trace_in_no_trace_lint = []
 [package.metadata.rust-analyzer]
 # This crate uses #![feature(rustc_private)]
 rustc_private = true
+
+# You might also need this in .vscode/settings.json
+# "rust-analyzer.check.command": "check",
+# "rust-analyzer.rustc.source": "discover",
```

Comment:
Can you please also update https://github.com/servo/servo/blob/main/docs/HACKING_QUICKSTART.md#visual-studio-code if necessary?

Reply:
This slows down rust-analyzer (that's why it isn't set by default) and it's is only really needed when working on crown, so it's mostly not needed (even I do not have it when not working on crown).